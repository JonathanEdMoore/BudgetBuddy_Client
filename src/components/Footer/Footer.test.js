import React from 'react'
import ReactDOM from 'react-dom'
import Footer from './Footer'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Footer smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><Footer /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('Footer snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
