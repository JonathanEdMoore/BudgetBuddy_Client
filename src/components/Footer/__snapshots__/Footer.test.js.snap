// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot test renders the UI as expected 1`] = `
<div>
  <footer
    role="content-info"
  >
    <Link
      to="/about"
    >
      About
    </Link>
    <span
      className="Hyph"
    >
       - 
    </span>
    <div>
      <SignupButton />
      <span
        className="Hyph"
      >
         - 
      </span>
      <LoginButton />
    </div>
  </footer>
</div>
`;
